packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/theme/design_system.dart

```diff
@@ -21,15 +21,25 @@ class DesignSystem extends ThemeExtension<DesignSystem> {
   DesignSystem.light()
       : colors = const DesignSystemColors.light(),
         icons = DesignSystemIcons(),
+        {{#enable_social_logins}}
+        images = const DesignSystemImages.light(),
```

Comment:
Let's have dark/light mode regardless if the social logins

Reply:
Done